src/EPR.Calculator.Frontend/Views/Shared/_ConfirmationPartial.cshtml

```diff
@@ -20,7 +20,10 @@
 
                 &nbsp; &nbsp;
                 <div class="govuk-button-group">
-                    <a href="@Url.Action("Index", @Model.RedirectController)" class="govuk-button" type="submit">@Model.SubmitText</a>
+                    @if (!string.IsNullOrEmpty(Model.SubmitText))
```

Comment:
Is it ok to change it to IsNullOrWhiteSpace please?

Reply:
Changed